test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,21 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() {
+        // Same objects no null
+        assertFalse(Utils.isAnyNull(1, 2, 3, 4, 5));
+
+        // Different objects no null
+        assertFalse(Utils.isAnyNull(1, "two", Integer.valueOf(3), Double.valueOf(4.0), "5"));
+
+        // Same objects with null
+        assertTrue(Utils.isAnyNull(1, 2, 3, 4, null));
```

Comment:
You can also try different number of arguments and different number of nulls.
eg. only one null object, only multiple null objects, no arguments etc.